lib/flipper/views/feature.erb

```diff
@@ -200,28 +200,19 @@
         </div>
       </div>
     </div>
-
-    <% if Flipper::UI.configuration.feature_removal_enabled %>
-    <div class="vads-l-row">
-      <div class="vads-l-col">
-        <div class="usa-alert usa-alert-warning vads-u-margin-bottom--5">
-          <div class="usa-alert-body">
-            <h4 class="usa-alert-heading"><%= Flipper::UI.configuration.delete.title %></h4>
-            <p>
-              <%= Flipper::UI.configuration.delete.description %>
-            </p>
-
-            <form action="<%= script_name %>/features/<%= @feature.key %>" method="post" onsubmit="return confirm('Are you sure you want to remove this feature from the list of features and disable it for everyone?')" class="usa-form">
-              <%== csrf_input_tag %>
-              <input type="hidden" name="_method" value="DELETE">
-              <button type="submit" name="action" value="Delete" class="vads-u-background-color--secondary vads-u-width--auto vads-u-margin--0 vads-u-margin-top--1" title="Remove feature from list of features and disable it.">Delete feature</button>
-            </form>
-          </div>
-        </div>
-      </div>
-    </div>
-    <% end %>
   </div>
 <% else %>
   <a href ="<%= SAML::URLService::VIRTUAL_HOST_MAPPINGS.dig(request.base_url, :base_redirect) %>">Sign in</a> with an identity-verified admin account to edit feature toggles.
 <% end %>
+
+<script type="text/javascript">
+  function lowercase_email() {
+      var srch = document.getElementById("actor-input");
+      var str = srch.value;
+      if (str.indexOf("@") > -1) {
+        srch.value = str.toLowerCase();
+      }
+      var form = document.getElementById("actors_form");
+      form.submit();
```

Comment:
Instead of changing the onsubmit behavior, what if we just ran this function `onchange` of the `<input>`?

I was going to suggest just doing `<input onchange: "this.value=this.value.toLowerCase()" ...>` but it seems you only want to do it if it has an `@`. It would simplify it quite a bit if we could remove that requirement.

Reply:
Thanks!  It's been a decade or more since I've written JavaScript.